services/shared/src/main/java/com/wedriveu/services/shared/entity/VehicleCounter.java

```diff
@@ -0,0 +1,83 @@
+package com.wedriveu.services.shared.entity;
+
+/**
+ * @author Stefano Bernagozzi
+ */
+
+public class VehicleCounter {
+    private int available;
+    private int broken;
+    private int booked;
+    private int recharging;
+    private int stolen;
+
+
+    public VehicleCounter() {
+        available = 0;
+        broken = 0;
+        booked = 0;
+        recharging = 0;
+        stolen = 0;
+    }
+
+    public void increaseAvailable() {
+        available++;
+    }
+
+    public void increaseBroken() {
+        broken++;
+    }
+
+    public void increaseBooked() {
+        booked++;
+    }
+
+    public void increaseStolen(){
+        stolen++;
+    }
+
+    public void increaseRecharging() {
+        recharging++;
+    }
+
+    public void decreaseAvailable() {
```

Comment:
Are all these `decrease` methods really needed for this task? See [https://it.wikipedia.org/wiki/You_aren%27t_gonna_need_it](https://it.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle